pkg/queue/postgres/retention_test.go

```diff
@@ -302,6 +315,22 @@ func TestAssertRetentionSchedule(t *testing.T) {
 			squirrel.Expr(`cron_schedule ~ '\d{1,2} * * * *'`),
 		}, "initial retention task not found")
 
+		specSQL := ""
+		expectedSQL := `DELETE FROM tasks WHERE status = 'finished' AND finished_at <= now() - interval '5.000000 minutes' AND queue = 'super_important' AND type = 'update-test'`
```

Comment:
I thought this one should not have used `now()` either.

Reply:
This now is not self-referential, i figured it was fine here

Reply:
Actually, I don' tthink this can change. This SQL is generated and then put into the DB, the SQL task then executes this sql, it doesn't generate the SQL per task, it is only generating it when the Schedule is made.